CMake/RealmCore.cmake

```diff
@@ -342,10 +391,15 @@ endfunction()
 
 function(clone_and_build_realm_sync branch)
     set(cmake_files ${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY})
+    if(REALM_PLATFORM STREQUAL "Android")
+        set(CONFIG_CMD, rm -f src/config.mk && REALM_CORE_PREFIX=${cmake_files}/realm-core/src/realm-core REALM_FORCE_OPENSSL=YES REALM_ENABLE_ENCRYPTION=YES REALM_ENABLE_ASSERTIONS= sh build.sh config && echo "ENABLE_ENCRYPTION    = yes" >> src/config.mk)
```

Comment:
I can't see how adding `ENABLE_ENCRYPTION` to `src/config.mk` could possibly be required. No variable of that name is ever referenced in the realm-sync codebase. Whether or not encryption is enabled is determined solely by core's configuration.